src/Generator/SqlGenerator.php

```diff
@@ -61,7 +62,9 @@ public function generate(
                 $query = $this->formatQuery($query);
             }
 
-            if ($nowdocOutput === true || ($nowdocOutput !== false && $formatted && strlen($query) > $maxLength )) {
+            if ($singleBlock === true) {
```

Comment:
`singleBlock` is typed as a boolean, you do not need to compare it with another boolean.
```suggestion
            if ($singleBlock) {
```